source/developers-guide/plugin-system/index.md

```diff
@@ -386,6 +386,26 @@ class SwagControllerExample extends Plugin
 }
 ```
 
+### Controller auto-registration
+
+As of version `5.2.7` there is no need for a subscriber which registers your controllers. This will be automatically done by the Shopware core.
+Just place a controller in `SwagControllerExample/Controllers/Backend|Frontend|Widgets|Api` and you are done. 
```

Comment:
To make use of it, create a file like `SwagControllerExample/Controllers/(Backend|Frontend|Widgets|Api)/YourControllerName.php` and it follow our controller naming conventions. After that, you'll be able to call `YourControllerName`.